x-pack/plugins/canvas/public/components/datatable/datatable.tsx

```diff
@@ -4,13 +4,14 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
-import React from 'react';
-import { EuiIcon, EuiPagination } from '@elastic/eui';
+import React, { FC } from 'react';
 import PropTypes from 'prop-types';
+import { EuiIcon, EuiPagination } from '@elastic/eui';
 import moment from 'moment';
 import { Paginate } from '../paginate';
+import { Datatable as DatatableType, DatatableColumn } from '../../../types';
 
-const getIcon = (type) => {
+const getIcon = (type: any) => {
```

Comment:
should type by `null | string` here?

Reply:
Good catch.  I got confused by the types in quotes below.  Fix coming.